docs/modeling/jointfitter.rst

```diff
@@ -0,0 +1,189 @@
+       area : float or `~astropy.units.Quantity`.
+           Integrated area
+           Note: amplitude = area / (stddev * np.sqrt(2 * np.pi))
+       mean : float or `~astropy.units.Quantity`.
+           Mean of the Gaussian.
+       stddev : float or `~astropy.units.Quantity`.
+           Standard deviation of the Gaussian with FWHM = 2 * stddev * np.sqrt(2 * np.log(2)).
+       """
+
+       area = Parameter(default=1)
+       mean = Parameter(default=0)
+
+       # Ensure stddev makes sense if its bounds are not explicitly set.
+       # stddev must be non-zero and positive.
+       stddev = Parameter(default=1, bounds=(FLOAT_EPSILON, None))
+
+       def bounding_box(self, factor=5.5):
+           """
+           Tuple defining the default ``bounding_box`` limits,
+           ``(x_low, x_high)``
+
+           Parameters
+           ----------
+           factor : float
+               The multiple of `stddev` used to define the limits.
+               The default is 5.5, corresponding to a relative error < 1e-7.
+           """
+
+           x0 = self.mean
+           dx = factor * self.stddev
+
+           return (x0 - dx, x0 + dx)
+
+       @staticmethod
+       def evaluate(x, area, mean, stddev):
+           """
+           Gaussian1D model function.
+           """
+           return (area / (stddev * np.sqrt(2 * np.pi))) * np.exp(
+               -0.5 * (x - mean) ** 2 / stddev ** 2
+           )
+
+       @property
+       def input_units(self):
+           if self.mean.unit is None:
+               return None
+           return {self.inputs[0]: self.mean.unit}
+
+Data to be fit is simulated.  The 1st spectral segment will have a spectral
+resolution that is a factor of 2 higher than the second segment.  The first
+segment will have wavelengths from 1 to 6 and the second from 4 to 10 giving
+an overlapping wavelength region from 4 to 6.
+
+.. code-block:: python
+
+   # Generate fake data
+   mean = 5.1
+   sigma1 = 0.2
+   sigma2 = 0.4
+   noise = 0.10
+
+   # compute the central amplitudes so the lines in each segement have the
+   # same area
+   area = 1.5
+   amp1 = area / np.sqrt(2.0 * math.pi * sigma1 ** 2)
+   amp2 = area / np.sqrt(2.0 * math.pi * sigma2 ** 2)
+
+   # segment 1
+   np.random.seed(0)
+   x1 = np.linspace(1.0, 6.0, 200)
+   y1 = amp1 * np.exp(-0.5 * (x1 - mean) ** 2 / sigma1 ** 2)
+   y1 += np.random.normal(0.0, noise, x1.shape)
+
+   # segment 2
+   np.random.seed(0)
+   x2 = np.linspace(4.0, 10.0, 200)
+   y2 = amp2 * np.exp(-0.5 * (x2 - mean) ** 2 / sigma2 ** 2)
+   y2 += np.random.normal(0.0, noise, x2.shape)
+
+Now define the models to be fit and fitter to use.  Then fit the two simulated
+datasets.
+
+.. code-block:: python
+
+   # define the two models to be fit
+   gjf1 = AreaGaussian1D(
+       area=1.0, mean=5.0, stddev=1.0, bounds={"area": (0, 10), "stddev": (0.1, 0.3)}
+   )
+   gjf2 = AreaGaussian1D(
+       area=1.0, mean=5.0, stddev=1.0, bounds={"area": (0, 10), "stddev": (0.1, 0.5)}
```

Comment:
I forget, do bounds on individual models have an effect when used with the JointFitter?

Reply:
No effect.  Checked and added the note near the beginning of this file to clearly state this.

Removed bounds definition to make this example cleaner.